### PR TITLE
feat: add environment variables for tag database and schema

### PR DESCRIPTION
**Pull request by arjunthangaraj**
This PR is for adding the environment variables for tag database and schema which should allow users to create the tag in different centralised location and use it in the models. This is for the issue mentioned [here](https://github.com/Montreal-Analytics/dbt-snowflake-utils/issues/28).

If not given any environment variables, the existing functionality will not get affected and tags will be created in the same database and schema as models.

README.md has been updated with the details and it is tested with the sample model on dbt v1.6.

### CODE REVIEW

**Comment by daniepett** [comment]
Hi @jamesweakley, any chance this could be reviewed? This feature would be really useful for us. Really appreciate the work you've done on this package.

**Comment by jamesweakley** [comment]
Hi @arjunthangaraj, apologies for the delay. The change looks good to me, thanks for contributing it.

I can't approve the merge but hopefully @JFrackson or another maintainer can.

**Comment by daniepett** [comment]
Hi @JFrackson, @mbhoopathy, @alfredodimassimo. Any chance we can get some reviews on this? :) Thanks

**Comment by arjunthangaraj** [comment]
Hi @JFrackson @alfredodimassimo @mbhoopathy , Could we get some reviews on this please? Thanks.

**Review by JFrackson — APPROVED** [review]
looks good to me!

**Comment by JFrackson** [comment]
Thanks for this change and sorry about the delay with the review! Merging and releasing now

**Pull request merged**
